package-lock.json

```diff
@@ -435,10 +435,11 @@
       }
     },
     "node_modules/@grpc/proto-loader": {
-      "version": "0.6.9",
-      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.9.tgz",
-      "integrity": "sha512-UlcCS8VbsU9d3XTXGiEVFonN7hXk+oMXZtoHHG2oSA1/GcDP1q6OUgs20PzHDGizzyi8ufGSUDlk3O2NyY7leg==",
+      "version": "0.6.12",
+      "resolved": "http://gv2-verdaccio-01.tools.exo.deindeal.ch:4873/@grpc%2fproto-loader/-/proto-loader-0.6.12.tgz",
```

Comment:
you must have some private npm proxy? please revert these. it's causing the build to fail and we also don't want to reference packages in some private mirror

Reply:
Sorry about it, I'll remove it